corehq/messaging/management/commands/export_conditional_alerts.py

```diff
@@ -0,0 +1,266 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+from corehq.apps.data_interfaces.models import (
+    AutomaticUpdateRule,
+    MatchPropertyDefinition,
+    CreateScheduleInstanceActionDefinition,
+)
+from corehq.messaging.scheduling.models import (
+    Schedule,
+    AlertSchedule,
+    TimedSchedule,
+    TimedEvent,
+    SMSContent,
+)
+from corehq.messaging.scheduling.scheduling_partitioned.models import CaseScheduleInstanceMixin
+from django.core.management.base import BaseCommand, CommandError
+from io import open as io_open
```

Comment:
The convention we've been using is `from io import open` (replace the built-in `open` in Python 2). I think that's a no-op in Python 3; @nickpell can confirm. In any case, it makes `open` work the same way on both Python 2 and 3. After doing that, all `open` calls should either specify binary mode or the `encoding` parameter.

Reply:
https://github.com/dimagi/commcare-hq/pull/21366/commits/4d8f89a83977fa6491262242a9e99a5370296adf